src/components/Editor.singleton.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Get instance of Editor component
+ * Using singleton approach here to avoid duplicate yjs import error
+ * @return {Promise<*>}
+ */
+export default async function getEditorInstance() {
+	if (!window.EditorInstance) {
+		if (window.EditorInstanceImporting) {
+			return await new Promise((resolve) => {
+				const intervalId = setInterval(() => {
+					if (!window.EditorInstance) {
+						return
+					}
+					resolve(window.EditorInstance)
+					clearInterval(intervalId)
+				}, 200)
+			})
+		} else {
+			window.EditorInstanceImporting = true
+		}
+		const Editor = await import(/* webpackChunkName: "editor" */'./Editor.vue')
+		window.EditorInstance = Editor.default
+	}
+	return window.EditorInstance
```

Comment:
In other app/libraries we use `_` to start the globals that should not be accessed directly. Maybe we can follow this pattern here and use those:
- `window._nc_text_editor_instance`
- `window._nc_text_editor_importing`